package.json

```diff
@@ -1,9 +1,8 @@
 {
 	"name": "skinview3d",
-	"version": "2.0.0-alpha.5",
+	"version": "2.0.1",
```

Comment:
Why change the version?

Reply:
I figured storybook being added could then be tracked to a tag version. If you don't agree with this approach I can revert it.

Reply:
Please revert it. I don't think it's a good idea to bump version number in PR.

Releasing should be done by running `npm version <new_version>`, which will create a new commit & a tag. Then you need to push the commit to master and run `npm publish`.